packages/core/processor.js

```diff
@@ -131,7 +132,11 @@ Processor.prototype = {
                 }
                 
                 return file.processed.then((result) => {
-                    file.exports = message(result, "classes");
+                    file.exports = Object.assign(
+                        Object.create(null),
+                        mapValues(file.values, (obj) => [ obj.value ]),
```

Comment:
Since `@value` statements don't have a composition graph to follow I don't think wrapping them in an array is worth it. This can probably mix in `file.values` directly instead.

Side benefit is that classes & `@value`s are easily differentiated just by glancing at them.

Reply:
Whoops, ignore that. Forgot that the output method expects everything to be an array so it can `.join` them. This still feels a bit weird, but it's easier to go w/ the flow on it than to add special logic elsewhere.

Reply:
Yeah, I played with changing the join function to check if it is an array. But it made more sense just to wrap the value in an array.